requirements/edx/base.txt

```diff
@@ -17,10 +17,10 @@ defusedxml==0.4.1
 django-babel-underscore==0.5.1
 markey==0.8  # From django-babel-underscore
 django-celery==3.2.1
-django-config-models==0.1.3
+django-config-models==0.1.8
 django-countries==4.0
 django-extensions==1.5.9
-django-filter==0.11.0
+django-filter==1.0.4
```

Comment:
I did *not* do that pass initially. But I just did a pass. Turns out the code imports DjangoFilterBackend from the wrong place, which I'll change. And `django_filters` is not in INSTALLED_APPS currently - I'll add it in.

Thank you for these tips!